src/app/api/common/ballots/getBallots.ts

```diff
@@ -0,0 +1,53 @@
+import { cache } from "react";
+
+async function getBallotsApi(roundId: string) {
+  const defaultPaginationMetadata = {
+    hasNext: false,
+    totalReturned: 1,
+    nextOffset: 1,
+  };
+
+  const defaultBallots = {
+    ballots: [
+      {
+        address: "0x",
```

Comment:
`address` in this case is the `submitterAddress`. We should unify these

Reply:
my read of the draft routes looked like these ballots will live on chain. if that's true, IMO we should include that address as part of the returned content when listing ballots. 

but if the address in the route itself is the submitter's address then i agree